frontend/pages/index.tsx

```diff
@@ -26,27 +28,34 @@ export default function Home({
         description={t('flathub-description')}
       />
       <div className='main-container'>
-        <h1>{t('apps-for-linux-right-here')}</h1>
-
-        <p
-          className='introduction'
-          style={{
-            marginBottom: '40px',
-            fontSize: '110%',
-            fontWeight: 300,
-            maxWidth: '700px',
-          }}
-        >
-          {t('welcome-to-flathub-index-text')}
-        </p>
-        <div style={{ "display": "flex", "gap": "12px" }}>
-          <a href='https://flatpak.org/setup/'>
-            <Button type='secondary'>{t('quick-setup')}</Button>
-          </a>
-          <Link href='/apps' passHref>
-            <Button type='secondary'>{t('explore')}</Button>
-          </Link>
+        <div style={{ "display": "flex", "gap": "12px", justifyContent: 'space-between' }}>
+          <div>
+            <h1>{t('apps-for-linux-right-here')}</h1>
+            <p
+              className='introduction'
+              style={{
+                marginBottom: '40px',
+                fontSize: '110%',
+                fontWeight: 300,
+                maxWidth: '700px',
+              }}
+            >
+              {t('welcome-to-flathub-index-text')}
+            </p>
+            <div style={{ "display": "flex", "gap": "12px" }}>
+              <a href='https://flatpak.org/setup/'>
+                <Button type='secondary'>{t('quick-setup')}</Button>
+              </a>
+              <Link href='/apps' passHref>
+                <Button type='secondary'>{t('explore')}</Button>
+              </Link>
+            </div>
+          </div>
+          <div className={styles.launchImage}>
+            <Image width='400' height="300px" src="/assets/landing.svg" alt="Landing page image" />
```

Comment:
I'd perhaps make the `alt` empty since this is just decorative

Reply:
Great catch, didn't know that's a thing https://www.w3.org/WAI/tutorials/images/decorative/